deltachat-ffi/src/lib.rs

```diff
@@ -1390,8 +1382,20 @@ pub unsafe extern "C" fn dc_get_msg(context: *mut dc_context_t, msg_id: u32) ->
             let message = match message::Message::load_from_db(ctx, MsgId::new(msg_id)) {
                 Ok(msg) => msg,
                 Err(e) => {
-                    error!(ctx, "Error getting msg #{}: {}", msg_id, e);
-                    return ptr::null_mut();
+                    if msg_id <= constants::DC_MSG_ID_LAST_SPECIAL {
+                        // C-core API returns empty messages, do the same
```

Comment:
good catch!
however, the returning of empty messages was not intenionally,i would regard the behavior of core-c as a bug here :)
but, it's probably the easiest way to emulate the behavior.